### PR TITLE
Internationalization: Translate page headers and Search dashboard actions

### LINKED ISSUE #60719: PageHeader & TabsBar not shown according to the language I set
**What happened**:
When I set the language to Chinese (in _grafana/conf/defaults. ini_，default_language = zh-Hans), 
PageHeader & TabsBar still show English

**What you expected to happen**:
When I change the default language, the PageHeader & TabsBar is displayed in that language

**How to reproduce it (as minimally and precisely as possible)**:
in _grafana/conf/defaults. ini_，set default_language = zh-Hans

**Anything else we need to know?**:
Can I submit a pr to fix this problem?
in _grafana/public/app/core/components/PageHeader/PageHeader.tsx_
`import { getNavTitle, getNavSubTitle } from 'app/core/components/NavBar/navBarItem-translations';`

**Environment**:
- Grafana version: The main branch
- Data source type & version: -
- OS Grafana is installed on: Local development
- User OS & Browser: Mac & Chrome
- Grafana plugins: -
- Others: -

### PR DESCRIPTION
**Pull request by TaitChan**
**What is this feature?**

PageHeader & PageHeaderTabs &  NavBarItem & DashboardActions & PanelMenu is displayed in default_language

**Why do we need this feature?**

When I set the default language to a different language (such as Chinese),
the main information area on the page was mixed in languages. 
I noticed that the navigation bar on the left has multiple languages, 
so the PageHeader should also have one.
The same goes for PanelMenu; 
I went through the code and found that some menus have multiple languages set and some don't.
I think this change will help international users have a better experience with grafana.

**Who is this feature for?**

All users who need to set the default language.

**Which issue(s) does this PR fix?**:

Fixes #60719

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by TaitChan** [comment]
Hi @joshhunt and @ashharrison90 , Please review my submitted, if has the need to change the place can tell me，Thanks♪(･ω･)ﾉ

File: public/app/core/components/PageHeader/PageHeader.tsx [hunk 1]
```diff
@@ -96,19 +97,18 @@ export const PageHeader: FC<Props> = ({ navItem: model, renderTitle, actions, in
   const renderHeader = (main: NavModelItem) => {
     const marginTop = main.icon === 'grafana' ? 12 : 14;
     const icon = main.icon && toIconName(main.icon);
-    const sub = subTitle ?? main.subTitle;
+    const sub = subTitle ?? getNavSubTitle(main.id) ?? main.subTitle;
+    const text = getNavTitle(main.id) ?? main.text;
```
Comment (joshhunt) [inline public/app/core/components/PageHeader/PageHeader.tsx:101]: nice! 👍🏻

**Review by joshhunt — COMMENTED** [review]
Great, thanks for this!

Just some changes needed to use `t()` directly in the few places I called out, and then run `yarn i18n:extract` to add them to the translation files 👍🏻

**Comment by TaitChan** [comment]
Hi @joshhunt , Thank you for your review. 😉
I have modified the use of `t()` according to your approval comments and run `yarn i18n:extract`. 
However, this command generates some json files, and I am not sure whether I have the permission to submit it, 
because in issues, I was reminded not to edit the files related to translation.
Looking forward to your reply.

**Review by joshhunt — APPROVED** [review]
Yup, this is fine. The JSON files need to be updated with placeholders for translations, but translations themselves cannot come from contributors.

**Comment by TaitChan** [comment]
Hi @joshhunt , I'm happy that the modified code passed your review, 
but I noticed that the `Enterprise PR check` module in the workflow failed. 
Is it because there is no tag added such as `enterprise-failed` or `enterprise-ok`?

**Comment by TaitChan** [comment]
> Hi @TaitChan - we're having some issues with external PRs with new checks that we're hoping to resolve shortly :)
> 
> Nothing for you (or me) to do right now, but I hope we can sort this out soon!

Hahaha, I understand. Hope there is good news 👌🏻

**Comment by TaitChan** [comment]
Hi @joshhunt , Thanks for merge, this is my first PR and I'm really happy to have it on grafana! 🎉

**Pull request merged**